.github/dependabot.yml

```diff
@@ -50,6 +50,22 @@ updates:
       interval: "weekly"
       day: "sunday"
       time: "16:00"
+    groups:
+      dependencies:
+        patterns:
+          - "@dependabot/yarn-lib"
+          - "@npmcli/arborist"
+          - "detect-indent"
+          - "nock"
+          - "npm"
+          - "@pnpm/lockfile-file"
+          - "@pnpm/dependency-path"
+          - "semver"
+      dev-dependencies:
+        patterns:
+          - "*eslint*"
```

Comment:
@landongrindheim would you want the same 'explicit' configuration to apply to dev-dependencies too? I can make the update too.

Reply:
I'm less concerned with the dev dependencies 😄